src/brevitas/graph/equalize.py

```diff
@@ -279,6 +394,26 @@ def _select_scale_computation_fn(
         raise RuntimeError(f"Scale computation type {scale_computation_type} not supported")
 
 
+class rotate_permute_mode:
+
+    def __init__(self, model, **kwargs):
+        self.rotation = GraphRotationEqualization(
+            **kwargs, use_parametrized_rotations=True, delay_rewriters=True)
+        self.model = model
+        self.rewriters = None
+
+    def __enter__(self):
+        model, rewriters = self.rotation.apply(self.model)
+        self.model = model
+        self.rewriters = rewriters
+        self.rotation.permute_class.setup_permute()
+
```

Comment:
```suggestion
        return self
```
to use rewriters outside the context.